### PR TITLE
Feature M2-954 Builder - Activity Items Tab - Item Configuration - Review stories Settings panel + Text input option

### PR DESCRIPTION
**Pull request by muhurov**

### CODE REVIEW

File: src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.styles.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    border-bottom: ${variables.borderWidth.md} solid ${variables.palette.outline_variant};
+  }
+`;
+
+export const StyledFormControl = styled(FormControl)`
+  width: 100%;
+  gap: 1.4rem;
+`;
+
+// export const StyledFormControlLabel = styled(FormControlLabel, shouldForwardProp)`
```
Comment (nmialik) [inline src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.styles.ts:64]: do we need this commented code?

File: src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
+import { useState, ChangeEvent } from 'react';
+import { FormLabel, FormGroup, FormControlLabel, Checkbox } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFormContext } from 'react-hook-form';
+
+import {
+  ItemConfigurationSettings,
+  DEFAULT_TIMER_VALUE,
+} from 'modules/Builder/features/ActivityItems/ItemConfiguration';
```
Comment (nmialik) [inline src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.tsx:9]: please use paths to specific file

File: src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.types.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { ItemConfigurationSettings } from 'modules/Builder/features/ActivityItems/ItemConfiguration';
```
Comment (nmialik) [inline src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.types.ts:1]: please use paths to specific file

File: src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.styles.ts (hunk 1 above)
Comment (muhurov) [reply src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.styles.ts]: fixed

File: src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.tsx (hunk 2 above)
Comment (muhurov) [reply src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.tsx]: fixed

File: src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.types.ts (hunk 3 above)
Comment (muhurov) [reply src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.types.ts]: fixed

**Review by nmialik — APPROVED** [review]

**Pull request merged**